Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.7.1-onbuild
+MAINTAINER Jimmy Mesta "jimmy.mesta@gmail.com"
+
+RUN wget https://dl.eff.org/certbot-auto -P /usr/local/sbin && \
```

Comment:
Removed Let's Encrypt support and using a self-signed cert only. We found that this app works well behind a LB / proxy and terminating TLS there with a valid certificate. I'm open to other suggestions though.